bin/run.sh

```diff
@@ -3,14 +3,13 @@
 if [ $# -lt 3 ]
 then
     echo "Usage:"
-    echo "./bin/run.sh two_fer ~/input/ ~/output/"
+    echo "./bin/run.sh two-fer ~/input/ ~/output/"
     exit 1
 fi
 
 problem_slug="$1"
 input_folder="$2"
 output_folder="$3"
-compiler_error_log="compiler.log"
 
 cp -r $input_folder /opt/test-runner/lib/$problem_slug
```

Comment:
The `/opt/test-runner/lib` directory won't be writable. I suggest to use the output folder for this.

Reply:
That does mildly complicate things because now I have to get the output folder into the test runner itself, but I can make that work. I'll update this once I have that working.